### ISSUE TITLE
create minified version on save when the related file does not exist

### ISSUE THREAD

**Issue by rrd108** [HookyQR/VSCodeMinify#29]
This is a feature request.

I think it would be nice if the plugin can create the minified versions on save when the related minified file does not yet exists.

I normally minify all my files, so with this I can skip the step when I have to create the first minified version manually.

**Comment by rrd108** [comment]
As I see and tested this ONLY works if you have ALREADY EXISTS. So the extension does not create a new file if it is not already there. Am I wrong?

**Comment by HookyQR** [comment]
Correct. You just have to run it manually once, then it will happen on save from then on. This avoids accidentally minifying files that you don't want/need minified.

**Comment by tiagocaus** [comment]
> Correct. You just have to run it manually once, then it will happen on save from then on. This avoids accidentally minifying files that you don't want/need minified.


- How to do it, if the file does not exist, automatically create the minified file.

- When created, updated, is it possible to send it via FTP automatically?
